Patches/Cursed Guns/Fedora.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+<Operation Class="PatchOperationFindMod">
+    <mods>
+      <li>Cursed Guns</li>
+    </mods>
+    <match Class="PatchOperationSequence">
+    <operations>
+
+	<li Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_FedoraCursedGun"]/statBases/StuffEffectMultiplierArmor</xpath>
+		<value>
+			<StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
+		</value>
+	</li>
+  
+	<li Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Apparel_FedoraCursedGun"]/apparel</xpath>
+		<value>
+			<layers>
+				<li>MiddleHead</li>

```

Comment:
The MiddleHead layer has been done away with, so this whole operation can just be removed.

Reply:
I'm still new to this kind of thing, so I'm not sure, if you're asking me to do something. Or how to do it.